ocaml/runtime/unix.c

```diff
@@ -17,6 +17,9 @@
 
 /* Unix-specific stuff */
 
+// XXX mshinwell for mshinwell: figure out what is going on here
+#include <sys/time.h>
```

Comment:
I've removed this hack and added back the upstream stanza of:
```
#ifdef HAS_GETTIMEOFDAY
#include <sys/time.h>
#endif
```
However I've put this after the include of `caml/config.h`.  I don't see otherwise how `HAS_GETTIMEOFDAY` is going to be defined (it's in `s.h` which is included from `caml/config.h`), and indeed I get a build failure on macOS without this fix.